e2e-polybft/framework/test-bridge.go

```diff
@@ -2,14 +2,30 @@ package framework
 
 import (
 	"fmt"
+	"os"
 	"path"
+	"path/filepath"
 	"strconv"
 	"testing"
 	"time"
 
 	"github.com/0xPolygon/polygon-edge/command/rootchain/server"
 )
 
+func init() {
+	wd, err := os.Getwd()
+	if err != nil {
+		return
+	}
+
+	parent := filepath.Dir(wd)
+	wd = filepath.Join(parent, "/artifacts/polygon-edge")
+	os.Setenv("EDGE_BINARY", wd)
+	os.Setenv("E2E_TESTS", "true")
+	os.Setenv("E2E_LOGS", "true")
+	os.Setenv("E2E_LOG_LEVEL", "debug")
+}
```

Comment:
Leftover, remove it.

```suggestion
```

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1258/commits/98b6e5890ea7c06fda67307fd63fe1387ddc36cd